sbom-utility-scripts/scripts/base-images-sbom-script/app/base_images_sbom_script.py

```diff
@@ -116,21 +188,28 @@ def main():
 
     args = parse_args()
 
-    base_images_from_dockerfile = args.base_images_from_dockerfile.read_text().splitlines()
-    base_images_digests = args.base_images_digests.read_text().splitlines()
+    with args.parsed_dockerfile.open("r") as f:
+        parsed_dockerfile = json.load(f)
+
+    base_images = get_base_images_from_dockerfile(parsed_dockerfile)
 
-    is_last_from_scratch = False
-    if base_images_from_dockerfile[-1] == "scratch":
-        is_last_from_scratch = True
+    base_images_digests_raw = args.base_images_digests.read_text().splitlines()
+    base_images_digests = {}
+    for item in base_images_digests_raw:
+        image_reference, digest = item.split("|")
+        base_images_digests[image_reference] = digest
```

Comment:
Can we shorten these lines as following?

```python
base_images_digests = dict(item.split("|") for item in base_images_digests_raw)
```

Reply:
updated